src/matrix_t/matrix_t.h

```diff
@@ -26,9 +27,10 @@ class matrix_t {
     channel = get_channel(data_type) | 0;
     _data_type_size.assign(
         {-1, 1, 4, -1, 4, -1, -1, -1, 8, -1, -1, -1, -1, -1, -1, -1, 8});
-    size = (cols * get_data_type_size(data_type) * channel) *
```

Comment:
I copied this code from `jsfeat` but size was refering to the ArrayBuffer size, a completely different things. If you use this code the array will not maintain its size but will enlarge unconsistently.

Reply:
Probably the `get_data_type_size` is not needed at all, I think i will delete it and related code.